gempython/tools/optohybrid_user_functions_xhal.py

```diff
@@ -334,15 +334,15 @@ def getTriggerSource(self):
         else:
             print("HwOptoHybrid.getTriggerSource() - No support for v3 electronics, exiting")
             sys.exit(os.EX_USAGE)
-    
+
     def getType(self):
         return (self.typeGEM,self.typeDet)
 
     def getVFATMask(self):
         """
         V3 electronics only
 
-        Returns a 24 bit number that should be used as the VFAT Mask
+        Returns a self.parentAMC.nVFAT bit number that should be used as the VFAT Mask
```

Comment:
[same comment to be applied everywhere](https://github.com/cms-gem-daq-project/cmsgemos/pull/287/files#r297225193)